lldb/source/Commands/CommandObjectThread.cpp

```diff
@@ -251,6 +251,9 @@ class CommandObjectThreadBacktrace : public CommandObjectIterateOverThreads {
       }
     }
 
+    thread->CalculateTarget()->GetStatistics().SetFirstBtTime(
+        thread->CalculateProcess(), *thread);
+
```

Comment:
What is this measuring? This will call the `SetFirstBtTime` and manually do a backtrace? What if the backtrace has already been done via the SB API? Won't this function go really quickly and not give us any real numbers?